README.md

```diff
@@ -2,9 +2,9 @@
 
 [![Build Status](https://travis-ci.com/dashevo/drive.svg?token=Pzix7aqnMuGS9c6BmBz2&branch=master)](https://travis-ci.com/dashevo/drive)
 
-> Dash network data storage backend service.
+> Distributed, decentralized application storage on the Dash p2p network.
 
-An [event-sourced](https://martinfowler.com/eaaDev/EventSourcing.html) metadata service built around the Command-Query Responsibility Segregation ([CQRS](https://martinfowler.com/bliki/CQRS.html)) pattern.
+Drive is the storage component of Dash Platform, allowing developers to store and secure their application data through Dash's masternode network. An application's current state is stored in the State View, which is a MongoDB instance hosted on a masternode. Historical application data is stored in IPFS and propagated to other nodes, thereby, ensuring data redundancy.
```

Comment:
I think the most important part is how data is replicated via chain. And then we can tell how we store data. Actually, you can take the diagram from my presentation. It would be cool.